modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,135 @@
+
+To create a child team in the Teams feature interface:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-teams.adoc[]
+
+. Select *Create Team*.
+. Enter a unique and intuitive name for your child team. +
+You can use only alphanumeric characters, hyphens, commas, periods, and spaces. The character limit is 255.
++
+. Select a parent team.
+. Select *Save*.
+
+The child team appears under its parent team in the list of teams. You can also select a parent team and navigate to the *Child teams* tab to see a list of all child teams under that parent team.
+
+== Manage Team Permissions
+
+When you begin using the Teams feature, your organization’s main team is named “Everyone at <Your Organization>” and does not have any permissions assigned.
+If you have the Organization Administrator permission, you can configure permissions for this team in addition to child teams.
+If you invite users to “Everyone at <Your Organization>” without assigning any permissions to that team, your users cannot execute tasks or view assets in your organization.
+
+If you have the Organization Administrator permission in a specific business group but not your entire organization, you can assign these permissions to any of existing teams or users within your organization.
+
+To configure permissions for teams in the Teams feature interface:
+
+include::partial$include-nav-steps-team-maintainer.adoc[]
+
+. Click the team for which you want to configure permissions.
+. Click the *Permissions* tab.
+. In the *Permissions* section, click *Add permissions*
+. In the *Select Permissions* window, select the permissions that you want to add, and then select *Select contexts*.
+. Select the business groups and environments to which you want to apply the permissions, and then select *Review*. +
+The permissions you selected appear alongside the business groups and environments to which they apply.
++
+. Select *Apply permissions*. +
+The permissions you selected appear in the *Permissions* section for your team.
+
+== Manage Team Users
+
+If you have the Organization Administrator permission in your root organization, you can add members to existing teams using three methods:
+
+* The Teams feature interface
+* Sending an invitation to a new user via email
+* External group mapping
+
+[[invite-users-with-teams]]
+=== Add Users using the Teams Interface
+
+include::partial$include-nav-steps-team-maintainer.adoc[]
+
+. In the *Teams* page, select the team to which you want to add a user.
+. Select *Add members*.
+. Select the name of each user you want to add to the team.
+. Select the membership status that you want to assign to each user: team member or team maintainer. + 
+A user who is added as a team maintainer receives team maintainer permissions as soon as you confirm their membership. 
++
+. Select *Add member*. +
+The user is added to the team and receives the team’s permissions.
```

Comment:
Ah, so...maybe not point out "existing" in Line 46?
And maybe this point should be made when we describe inheritance?

Reply:
Why do we specify permission reception here but not for other two methods?